manifests/stage-ipoib-cni/0040_scc.openshift.yaml

```diff
@@ -0,0 +1,49 @@
+{{if eq .RuntimeSpec.OSName "rhcos"}}
+# Please edit the object below. Lines beginning with a '#' will be ignored,
```

Comment:
nit: For me it is a bit easier to read specs if yaml keys are ordered with k8s style:
apiVersion:
kind:
metadata:
..other fields

WDYT?

Reply:
This appears to be a common format for OCP Security Context Constraints: https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html
We use the same format for all other states.
We can change but for consistency it needs to be done for all similar files in the project

Reply:
Ok, let's keep it as is